subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -1002,7 +1002,9 @@ void ticker_worker(void *param)
 		 * with other ticker nodes
 		 */
 		if (ticker->ticks_slot != 0U &&
-		   (slot_reserved || ticker_resolve_collision(node, ticker))) {
+		   (slot_reserved ||
+		    (instance->ticks_slot_previous > ticks_expired) ||
```

Comment:
I would prefer to move the expression up and set `slot_reserved` for clarity.

Reply:
In the current implementation if `slot_reserved` is true, then rest of the expression is not evaluated. If I move the expression up then `instance->ticks_slot_previous > ticks_expired` will be evaluated irrespective of `slot_reserved` being true.